### PR TITLE
[JENKINS-58157] keep fixed length for the delete build button

### PR DESCRIPTION
**Pull request by mamh2021**
@jsoref 
@oleg-nenashev 

donot add build displayName to the "delete build" button

fix "delete build" button too long,https://issues.jenkins-ci.org/browse/JENKINS-58157

this issue come from this commit: [JENKINS-55848] Identify the name of the thing being deleted (#3871)
revert  core/src/main/resources/hudson/model/Run/delete.jelly
revert  core/src/main/resources/hudson/model/Run/delete.properties

Signed-off-by: Mamh-Linux <bright.ma@blackshark.com>

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

### Proposed changelog entries

* Entry 1: Issue, Human-readable Text
* ...

### Submitter checklist

- [x] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Other localizations may need an update though it is better to have more feedback from reviewers first

**Comment by mmh891113** [comment]
> Other localizations may need an update though it is better to have more feedback from reviewers first

this patch just revert the two files. there is no impact the localizations. the localizations keep the same as before.
I test EN and zh_CN,zh_TW it show OK.

**Review by daniel-beck — CHANGES_REQUESTED** [review]
This negates the usability improvements of the original change.

I would consider it rather unlikely that display names are very long in general, and the problem isn't a severe one when it occurs, so I'd rather see a proper solution, perhaps involving label trimming, rather than a simple revert.

**Comment by mmh891113** [comment]
> This negates the usability improvements of the original change.

This revert just keep "delete build" button  the previous.
I think It's better to keep  the side-panel  short.

File: core/src/main/resources/hudson/model/Run/delete.jelly [hunk 1]
```diff
@@ -28,6 +28,6 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
   <j:if test="${!it.building}">
-    <l:task href="${buildUrl.baseUrl}/confirmDelete" icon="icon-edit-delete icon-md" permission="${it.DELETE}" title="${%delete.build(it.displayName)}"/>
+    <l:task href="${buildUrl.baseUrl}/confirmDelete" icon="icon-edit-delete icon-md" permission="${it.DELETE}" title="${%Delete this build}"/>
   </j:if>
-</j:jelly>
+</j:jelly>
```
Comment (LinuxSuRen) [inline core/src/main/resources/hudson/model/Run/delete.jelly]: We usually keep the last blank line

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Please keep sync with other localization plugins if you want to make a change about the localization.

**Comment by mmh891113** [comment]
i force push to my master close this PR......

**Pull request closed without merging**